File: ft_home/src/main/java/com/wuc/ft_home/view/mine/MineFragment.java
package com.wuc.ft_home.view.mine;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import com.wuc.ft_home.R;

/**
 * @author: wuchao
 * @date: 2019-09-17 10:39
 * @desciption: 首页我的Fragment
 */
public class MineFragment extends Fragment {

  public static MineFragment newInstance() {
    MineFragment fragment = new MineFragment();
    Bundle args = new Bundle();
    fragment.setArguments(args);
    return fragment;
  }

  @Override
  public void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);

  }

  @Override
  public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
    return inflater.inflate(R.layout.fragment_mine_layout, container, false);
  }

  @Override
  public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
    super.onViewCreated(view, savedInstanceState);
    //发请求更新UI
  }

}

File: app_voice/src/main/java/com/wuc/voice/view/mine/MineFragment.kt
package com.wuc.voice.view.mine


import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.wuc.voice.R

/**
 * @author:     wuchao
 * @date:       2019-09-17 23:02
 * @desciption: 首页 我的Fragment
 */
class MineFragment : Fragment() {

    companion object {
        fun newInstance(): Fragment {
            return MineFragment()
        }
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_mine_layout, container, false)
    }


}
